shared/src/commonMain/kotlin/org/hyperskill/app/notification_onboarding/injection/NotificationsOnboardingComponentImpl.kt

```diff
@@ -0,0 +1,17 @@
+package org.hyperskill.app.notification_onboarding.injection
+
+import org.hyperskill.app.core.injection.AppGraph
+import org.hyperskill.app.notification_onboarding.presentation.NotificationsOnboardingFeature.Action
+import org.hyperskill.app.notification_onboarding.presentation.NotificationsOnboardingFeature.Message
+import org.hyperskill.app.notification_onboarding.presentation.NotificationsOnboardingFeature.State
+import ru.nobird.app.presentation.redux.feature.Feature
+
+class NotificationsOnboardingComponentImpl(
```

Comment:
Since we have `BaseAppGraph` with a common implementation -> we can make component implementation also internal